sphinxcontrib/katex.py

```diff
@@ -74,31 +75,19 @@ def get_latex(node):
         return node.astext()  # for Sphinx >= 1.8.0
 
 
+def run_katex(latex):
+    p = subprocess.Popen(('katex', ), stdout=subprocess.PIPE, stdin=subprocess.PIPE)
+    stdout, stderr = p.communicate(latex.encode('utf-8'))
+    return stdout.decode('utf-8')
+
+
 def html_visit_math(self, node):
-    self.body.append(self.starttag(node, 'span', '', CLASS='math'))
-    self.body.append(self.builder.config.katex_inline[0] +
-                     self.encode(get_latex(node)) +
-                     self.builder.config.katex_inline[1] + '</span>')
+    self.body.append(run_katex(get_latex(node)))
```

Comment:
I think I need to pass some extra arguments to katex to make it do inline or block mode

Reply:
It might be that for inline math no option is needed.
For display mode (as used in the next function) the option `--display-mode` has to be used.